src/problem/mod.rs

```diff
@@ -123,6 +125,9 @@ pub enum Problem {
     NewTopLevelPackageShouldBeByNameWithCustomArgument(
         npv_163::NewTopLevelPackageShouldBeByNameWithCustomArgument,
     ),
+
+    /// NPV-170: by-name package cannot be number prefixed
+    ByNamePackegPrefixedWithNumber(npv_170::ByNamePackegPrefixedWithNumber),
```

Comment:
typo: ByNamePackegPrefixedWithNumber -> ByNamePackagePrefixedWithNumber

LSP rename could do it faster and also rename where it is used